### DISCUSSION TITLE
Integrate browser via headless remote isolated browser

### DISCUSSION THREAD

**Discussion by crisdosaygo** [DustinBrett/daedalOS#138] [Ideas]
You can use my project if you think it's not a terrible idea...

https://github.com/crisdosyago/BrowserBox

**Comment by DustinBrett** [comment]
Thanks for the suggestion! I am trying to avoid anything back-end at the moment, but in the future when I begin work on brain.dustinbrett.com I will start work on backend and maybe at that point.

**Comment by DustinBrett** [comment]
Actually I am not even sure I understand, is that what this is? How would you imagine I'd integrate it?

**Comment by crisdosaygo** [comment]
No worries. Yes this would require a backend to run the browser on, the frontend BrowserView component is just an interactive window to the browser which runs in the server. Effectively it becomes an unrestricted iframe with browser UI controls, similar to a WebView in mobile or Electron.

**Reply by DustinBrett** [reply]
Ah ok, thanks for clarifying. I'll keep trying to find a way to integrate it.
